metadata_service/api/user.py

```diff
@@ -31,8 +36,211 @@ def __init__(self) -> None:
 
     def get(self, user_id: str) -> Iterable[Union[Mapping, int, None]]:
         try:
-            table = self.neo4j.get_user_detail(user_id=user_id)
-            return marshal(table, user_detail_fields), HTTPStatus.OK
+            user = self.neo4j.get_user_detail(user_id=user_id)
+            return marshal(user, user_detail_fields), HTTPStatus.OK
 
         except NotFoundException:
             return {'message': 'User id {} does not exist'.format(user_id)}, HTTPStatus.NOT_FOUND
+
+
+class UserBookmarkAPI(Resource):
+    """
+    Build get / put API to support user bookmark resource features.
+    It will create a relationship(follow / followed_by) between user and resources(table, dashboard etc)
```

Comment:
I wonder the difference between bookmark and follow. If it's the same feature, how about unifying the term?